code/controllers/Processes/fast_machinery.dm

```diff
@@ -0,0 +1,57 @@
+//Faster version of the one in machinery.dm
+
+var/global/list/fast_machines = list()
+/datum/controller/process/fast_machinery
+	schedule_interval = 10 // every 1 second.
```

Comment:
Drop it to 0.5 seconds, it works without lag if you're going at 25 moved per second.